samples/mongodb/simple/sample.md

```diff
@@ -10,54 +10,66 @@ redirects:
 - samples/mongodb-tekmaven/databus
 ---
 
-
 ## Prerequisites
 
-Ensure an instance of [MongoDB](https://www.mongodb.com/) is running on `localhost:27017`. See [Install MongoDB on Windows](https://docs.mongodb.com/manual/tutorial/install-mongodb-on-windows/).
+Ensure an instance of [MongoDB](https://www.mongodb.com/) is executing on `localhost:27017`.
+
+The easiest way to do this is to run MongoDB in Docker by running the following commands
+
+```shell
+docker run -d -p 27017:27017 --name TestMongoDB mongo:latest --replSet tr0
```

Comment:
`--replSet tr0` does the trick right?

Reply:
I guess that sets the instance to be part of a replica set, and the command below intitiates the replica set.

Reply:
Correct